book.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Just like the first reviewer has said, rental should be marked with` attr_accessor`, as we intend to both access (get) as well as write (set) it. Kindly ensure to improve upon this aspect. It seems you missed implementing this change. Kindly do so